extension-config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "version": "1.0.0",
+  "id": "@shopgate-project/sticky-pdp-media",
+  "components": [
+    {
+      "id": "StickyMedia",
+      "path": "frontend/portals/StickyMedia/index.jsx",
+      "target": "component.product-media-section",
+      "type": "portals"
+    }
+  ],
+  "configuration": {
+    "scrolledTopOffset": {
+      "type": "admin",
+      "destination": "frontend",
+      "default": "-100vw",
```

Comment:
`vw` viewport width? Or it should be a height `-100vh`?

Reply:
width is correct since the height of the image depends on the width of the screen, not the height.